README.md

```diff
@@ -75,6 +75,31 @@ make -C out/ -j9
 sudo make install-linux
 ```
 
+### FreeBSD
+
+```bash
+# Clone this repo
+git clone https://github.com/nodejs/llnode.git && cd llnode
+
+# Install llvm with lldb and headers 
+pkg install llvm39
+rm -f /usr/bin/lldb
+ln -s /usr/local/bin/lldb39 /usr/bin/lldb
+
+# Initialize GYP
+git clone https://chromium.googlesource.com/external/gyp.git tools/gyp
+
+# Configure
+./gyp_llnode -Dlldb_dir=/usr/local/llvm39/
+
+# Build
+gmake -C out/ -j9
+
+# Install
+ComputeSystemPluginsDirectory is not implemented in FreeBSD
+The llnode.so will have to be loaded manually 
```

Comment:
I've redone the README.md changes as in the meantime one of my changes re-organised it so these didn't merge. I'll sort out the merge for this today.